sections/07_conclusions.md

```diff
@@ -91,22 +91,41 @@ very large collections of input data into interpretable models that spur
 scientists to ask questions that they didn't know how to ask, it will be clear
 that deep learning has transformed biology and medicine.
 
+### Continuous collaborative manuscript drafting
```

Comment:
See what I did in https://github.com/greenelab/deep-review/pull/477/commits/11276830059ba1f740520ac1bfeb7cd54332fb7c. I'm open to other rearrangements like what @agitter suggested if there's a strong preference. Let's deal with that in another PR if desired. Will merge this now.